tasmota/my_user_config.h

```diff
@@ -395,7 +395,9 @@
                                                  // Enabled by default on ESP32 and variants
 
 // -- ESP-NOW -------------------------------------
-//#define USE_TASMESH                              // Enable Tasmota Mesh using ESP-NOW (+11k code)
+#define USE_TASMESH                              // Enable Tasmota Mesh using ESP-NOW (+11k code)
+#define TASMESH_HEARTBEAT      1                 // If enabled, the broker will detect when nodes come online and offline and send Birth and LWT messages over MQTT correspondingly (0 = off, 1 = on)
+#define TASMESH_OFFLINE_DELAY  3                 // Maximum number of seconds since the last heartbeat before the broker considers a node to be offline
```

Comment:
Please change to `#define USE_TASMESH_OFFLINE_DELAY  3`

Reply:
Configuration values generally do not have the `USE_` prefix

Reply:
Ah ok. This are my own conventions then. Still I prefer to have the same prefix for all, it makes searches easier